src/converters/investimentalConverter.test.ts

```diff
@@ -0,0 +1,125 @@
+import { InvestimentalConverter } from "./investimentalConverter";
+import { SecurityService } from "../securityService";
+import { GhostfolioExport } from "../models/ghostfolioExport";
+import YahooFinanceServiceMock from "../testing/yahooFinanceServiceMock";
+
+
+describe("investimentalConverter", () => {
+
+  beforeEach(() => {
+    jest.spyOn(console, "log").mockImplementation(jest.fn());
+  });
+
+  afterEach(() => {
+    jest.clearAllMocks();
+  });
+
+  it("should construct", () => {
+
+    // Act
+    const sut = new InvestimentalConverter(new SecurityService(new YahooFinanceServiceMock()));
+
+    // Assert
+    expect(sut).toBeTruthy();
+  });
+
+  it("should process sample CSV file", (done) => {
+
+    // Arange
+    const sut = new InvestimentalConverter(new SecurityService(new YahooFinanceServiceMock()));
+    const inputFile = "samples/investimental-export.csv";
+
+    // Act
+    sut.readAndProcessFile(inputFile, (actualExport: GhostfolioExport) => {
+
+      // Assert
+      expect(actualExport).toBeTruthy();
+      expect(actualExport.activities.length).toBeGreaterThan(0);
+      expect(actualExport.activities.length).toBe(6);
```

Comment:
Given the length of the sample CSV, are you sure this number is correct? There are 51 records in the sample file, does it really "skip" 45 of them?

Reply:
Yes, I'm sure.
There are lots of records but most of then have Update type `Chg` and we don't need these in Ghostfolio.

Update type `New` - means a new order is created
Update type `Chg` - means an order has been updated
Update type `Fil` - means the order has been executed